### PR TITLE
mco: add optional e2e-aws-upgrade test for single-node

### PR DESCRIPTION
**Pull request by sinnykumari**

### CODE REVIEW

**Comment by sinnykumari** [comment]
Since new jobs has been added, these failing tests are expected.

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml [hunk 1]
```diff
@@ -517,6 +517,71 @@ presubmits:
...
+    branches:
+    - release-4.9
+    cluster: build02
+    context: ci/prow/e2e-aws-upgrade-single-node
+    decorate: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-machine-config-operator-release-4.9-e2e-aws-upgrade-single-node
+    rerun_command: /test e2e-aws-upgrade-single-node
```
Comment (kikisdeliveryservice) [inline ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml:531]:
```suggestion
    optional: true
    rerun_command: /test e2e-aws-upgrade-single-node
```

**Review by kikisdeliveryservice — CHANGES_REQUESTED** [review]
one change.

also, will this be a periodic or something as well? I hope it's not just going to live in the mco repo?

**Comment by sinnykumari** [comment]
> also, will this be a periodic or something as well? I hope it's not just going to live in the mco repo?

Yes, it is already running in the periodic as well https://prow.ci.openshift.org/job-history/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.9-e2e-aws-upgrade-single-node .

Reason of adding it in MCO repo as well is to make sure we don't regress due to any MCO changes when we initiate reboot during upgrade.

File: ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml (hunk 1 above)
Comment (sinnykumari) [reply ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-release-4.9-presubmits.yaml:532]: good catch! fixed.

**Review by kikisdeliveryservice — APPROVED** [review]
thanks for the change, this now looks good!
/lgtm

**Pull request merged**